tests/test_order.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+import rtoml
+
+"""
+When testing preserving of order we just check if
+the stringified TOML dictionary is in the same sequence
+as the original TOML string.
+"""
+
+
+@pytest.mark.parametrize(
+    'input_toml,output_toml',
+    [
+        (
+            """
+something = true
+lion = 'aslan'
+""",
+            "{'something': True, 'lion': 'aslan'}",
```

Comment:
```suggestion
            {'something': True, 'lion': 'aslan'},
```
No reason for this to be a string.

Reply:
When `expected_output` is a dictionary instead of a string and later on I do:
```
rtoml.load(input_toml) == expected_output
```
then I have no real test as I just compare two dictionaries where the sequence of the dict elements do not matter. Or is my thinking wrong?

Reply:
Humm, good point, but I still think it would be better not to be comparing strings.

Better to do

```py
loaded = rtoml.load(input_toml)
assert loaded == expected_output
assert list(loaded.items() == list(expected_output.items())  # check order  is maintained

Reply:
I agree fully. Thanks for the suggestion which should do nicely.